src/observability/publisher.go

```diff
@@ -41,7 +50,51 @@ func ProcessSystemSummary() {
 			// clear each published entry from data map
 			delete(tempSummarizerMap, ss)
 		}
+		log.Info().Msgf("Published %v events in %v", count, time.Since(timeVar))
 		PublisherMutex.Unlock()
 	}
 	ProcessSystemSummaryWg.Wait()
 }
+
+func getAggregatedSummaryMap(tempSummarizerMap map[types.SystemSummary]types.SysSummaryTimeCount) map[types.SystemSummary]types.SysSummaryTimeCount {
+
+	var fileArr []string
+	log.Info().Msgf("Events before aggregation: [%v]", len(tempSummarizerMap))
+
+	fileSumMap := make(map[types.SystemSummary]types.SysSummaryTimeCount)
+	updatedSummarizerMap := make(map[types.SystemSummary]types.SysSummaryTimeCount)
```

Comment:
Where do we clear this updatedSummarizerMap ?

Reply:
Since this map is local to getAggregatedSummaryMap(), I assume that every time the function gets called it'll automatically get recreated. Do you think we need to clear it separately?